limits/aio/storage/redis.py

```diff
@@ -96,6 +106,31 @@ async def get_moving_window(
             return float(window[0]), window[1]  # type: ignore
         return timestamp, 0
 
+    async def get_sliding_window(
+        self, key: str, expiry: Optional[int] = None
+    ) -> Tuple[int, float, int, float]:
+        if expiry is None:
+            raise ValueError("the expiry value is needed for this storage.")
+        previous_key = self.prefixed_key(self._previous_window_key(key))
+        current_key = self.prefixed_key(self._current_window_key(key))
+
+        window = await self.lua_sliding_window.execute(
+            [previous_key, current_key], [expiry]
+        )
+        if window:
+            previous_count, previous_expires_in, current_count, current_expires_in = (
+                int(window[0] or 0),  # type: ignore
+                max(0, float(window[1] or 0)) / 1000,  # type: ignore
+                int(window[2] or 0),  # type: ignore
+                max(0, float(window[3] or 0)) / 1000,  # type: ignore
+            )
+        return (
```

Comment:
These could end up being uninitialized

Reply:
I've added a default value.

Sync version:

```python
if window := self.lua_sliding_window([previous_key, current_key], [expiry]):
    return (
        int(window[0] or 0),
        max(0, float(window[1] or 0)) / 1000,
        int(window[2] or 0),
        max(0, float(window[3] or 0)) / 1000,
    )
return 0, float(0), 0, float(0)
```
Async version:
```python
if window:
    return (
        int(window[0] or 0),  # type: ignore
        max(0, float(window[1] or 0)) / 1000,  # type: ignore
        int(window[2] or 0),  # type: ignore
        max(0, float(window[3] or 0)) / 1000,  # type: ignore
    )
return 0, float(0), 0, float(0)
```

Is that ok?

Reply:
looks good! Thank you!